src/assets/locales/de/translations.json

```diff
@@ -114,9 +114,9 @@
   "about-us.leadbox.title": "Haben Sie Interesse bekommen?",
   "about-us.leadbox.link": "Karriere",
   "services.hero": "Leistungen",
-  "services.title": "Unser Angebot an Sie",
+  "services.title": "Wir verbinden Development & Design",
```

Comment:
This is different to `Wir verbinden. Mit Technologie.` (via `main.services.title"`) which we could just reuse here as on the start page?